streams/runners/storm/runtime/src/test/java/com/hortonworks/streamline/streams/runtime/AvroStreamsSnapshotDeserializerTest.java

```diff
@@ -66,7 +68,10 @@ public void testAvroPayloadConversions() throws Exception {
 
             new Expectations() {
                 {
-                    mockSchemaRegistryClient.getSchemaMetadataInfo(anyLong);
+                    mockSchemaRegistryClient.getSchemaVersionInfo(withEqual(SCHEMA_ID_VERSION));
+                    //only versions of schema matters for this mock, as getVersion is only used during de-serialization
+                    result = new SchemaVersionInfo(1l, schemaMetadata.getName(), SCHEMA_ID_VERSION.getVersion(), "doesNotMatter",1l, "doesNotMatter");
+                    mockSchemaRegistryClient.getSchemaMetadataInfo(withEqual(schemaMetadata.getName()));
```

Comment:
Not that important really. Its just a little stricter than before. It just makes sure that its called with same schema name as the one used for serialization.